### PR TITLE
Feature/console paging

### PR DESCRIPTION
**Pull request by Potentatus**
Added paging in console

### CODE REVIEW

File: ConsoleApp/ForumAppRunner.cs [hunk 1]
```diff
@@ -28,6 +28,10 @@ public ForumAppRunner(IServiceProvider serviceProvider, IConsoleReader consoleRe
 
         private int CallCommand(params string[] args)
         {
+            if(args.Length == 0)
```
Comment (progala2) [inline ConsoleApp/ForumAppRunner.cs]: Why do you added it?
It should work for no arguments.

File: ConsoleApp/ConsoleContext.cs [hunk 2]
```diff
@@ -51,33 +52,48 @@ public void ShowMessages()
...
-                _writer.WriteLabel($"{emptySpace} Date:");
-                _writer.WriteLine($"[{node.CreationDate}]");
-
-                _writer.WriteLabel($"{emptySpace} Text: ");
-                _writer.WriteLine(ShowResponses
-                    ? $"{node.Text}"
-                    : $"{(node.Text.Length > TextMsgShortLength ? $"{node.Text.Substring(0, TextMsgShortLength)}(..)" : node.Text)}");
-
-                if (!HideTags)
+                var input = System.Console.ReadLine();
```
Comment (progala2) [inline ConsoleApp/ConsoleContext.cs]: Use IConsoleReader instead

File: ConsoleApp/ForumAppRunner.cs (hunk 1 above)
Comment (progala2) [reply ConsoleApp/ForumAppRunner.cs]: The help should be written on the console when someone didn't pass correct arguments. It's good behavior.
with this return explore command will stop working (you can use it without arguments)

Comment (Potentatus) [reply ConsoleApp/ForumAppRunner.cs]: I do what I do and that's all what I do - I'm Popey The CoderMan xD
(as discussed earlier it prevents showing whole "usage" everytime user push empty enter)

**Pull request merged**